parser.go

```diff
@@ -358,6 +336,8 @@ func ArrayEach(data []byte, cb func(value []byte, dataType ValueType, offset int
 		if data[offset] != ',' {
 			return MalformedArrayError
 		}
+
+		offset++
```

Comment:
`offset++` to replicate the old functionality of `nextToken` with `skipComma` on. This is cheaper and safer because it skips exactly one comma, rather than potentially many commas.